Cabal/doc/developing-packages.rst

```diff
@@ -1,83 +1,119 @@
 Quickstart
 ==========
 
-Lets assume we have created a project directory and already have a
-Haskell module or two.
+Starting from scratch, we're going to walk you through creating a simple
+Haskell project.
```

Comment:
One have to be very careful with words *project* and *package*.

This document doesn't create `cabal.project` file, so the *word* project should not be used at all. Alternatively one have to describe how project and package relate.

As this is page is targeted to beginners, we should not teach them anything we need then unteach later.